### DISCUSSION TITLE
Who can tell me about this project?

### DISCUSSION THREAD

**Discussion by M-Enes** [Naruno/Naruno#76] [Q&A]
I'm a very new contributor of this project. I want to learn Decentra-Network project system basics and how to work. Thank you.

**Comment by onuratakan** [comment]
Hi, welcome.

Decentra Network is an open source Blockchain Network.

THE Decentra Network
- fully secure (Unhackable)
- high scalable
- energy friendly
- super fast
- fully decentralized (no centralization trend)
- low size
Blockchain Network


And in Turkey  the first
- Energy and environment friendly
- Non-centralized trend
- Providing protection against Sybil attacks (51 attacks etc.)
- Can work on all hardware
Blockchain



Decnetra Network have a different structure. But ı can explain as a basic:

* Blockchain folder:
This folder have a two subfolder, this is Block and Candidate Block
  - Block
    The Lots of Block Function in here, for example, Calculating The Block Hash, Getting the block from db, save the block to the Blockchain db, Validate and create the new block etc
  - Candidate Block
    We use the FBA (Federated Byzantine Aggrement) consensus algorithm therefore we need a candidate block every block, and we process the candidate blocks in here

* Consensus Folder
  The consensus is most important of the blockchain networks, in this folder we have a 3 file, this is consensus main, consensus first round, consensus second round
  - consensus main 
    The main consensus process controller in here, this is control the status and routing the first/second round or close the block.
  - consensus first round
    In here we analysing, sending the candidate block andnd end of this round we select the transaction which will into the current block.
  - consensus second round
    In here we analysing, sending the candidate block hash and we confirm the consensus. (if 80% of the candidate blocks have a same hash blocks is confirmed )

Other folders have a some class controller and mechanism.


We share a issue with "good first issue" label you cand do this, is simple and we write (how is it going to be made)

Example [this](https://github.com/Decentra-Network/Decentra-Network/issues/59) issue need one line, you can do this, and ı can help you.

Sorry for my English, I'm still learning. =)

Lastly you can ask me anything.
